show-portal-user-info/src/main/java/com/esri/arcgismaps/sample/showportaluserinfo/screens/MainScreen.kt

```diff
@@ -0,0 +1,232 @@
+import androidx.compose.ui.text.input.KeyboardType
+import androidx.compose.ui.unit.dp
+import androidx.lifecycle.viewmodel.compose.viewModel
+import com.arcgismaps.toolkit.authentication.AuthenticatorState
+import com.arcgismaps.toolkit.authentication.DialogAuthenticator
+import com.esri.arcgismaps.sample.sampleslib.components.SampleTopAppBar
+import com.esri.arcgismaps.sample.showportaluserinfo.components.AuthenticationAppViewModel
+
+/**
+ * Main screen layout for the sample app
+ */
+@Composable
+fun MainScreen(sampleName: String, application: Application) {
+
+    val authenticationAppViewModel = viewModel { AuthenticationAppViewModel(application) }
+    val authenticatorState: AuthenticatorState = authenticationAppViewModel.authenticatorState
+
+    Scaffold(
+        topBar = { SampleTopAppBar(title = sampleName) },
+        content = {
+            Column(
+                modifier = Modifier
+                    .fillMaxSize()
+                    .padding(it)
+            ) {
+                val infoText = authenticationAppViewModel.infoText.collectAsState().value
+                val isLoading = authenticationAppViewModel.isLoading.collectAsState().value
+                PortalDetails(
+                    url = authenticationAppViewModel.url.collectAsState().value,
+                    onSetUrl = authenticationAppViewModel::setUrl,
+                    onSignOut = authenticationAppViewModel::signOut,
+                    onLoadPortal = authenticationAppViewModel::loadPortal
+                )
+                InfoScreen(infoText = infoText,
+                    username = authenticationAppViewModel.portalUserName.collectAsState().value,
+                    email =  authenticationAppViewModel.emailID.collectAsState().value,
+                    creationDate = authenticationAppViewModel.userCreationDate.collectAsState().value,
+                    portalName = authenticationAppViewModel.portalName.collectAsState().value,
+                    userThumbnail = authenticationAppViewModel.userThumbnail.collectAsState().value,
+                    isLoading = isLoading)
+            }
+            DialogAuthenticator(authenticatorState = authenticatorState)
+        }
+    )
+}
+
+/**
+ * Allows the user to enter a url and load a portal.
+ * It uses OAuth under the hood, and has a button to clear credentials.
+ *
+ * @param url the string url to display in the text field
+ * @param onSetUrl called when the url should be changed
+ * @param onSignOut called when any stored credentials should be cleared
+ * @param onLoadPortal called when the [url] should be loaded
+ * @since 200.2.0
+ */
+@OptIn(ExperimentalComposeUiApi::class)
+@Composable
+private fun PortalDetails(
+    url: String,
+    onSetUrl: (String) -> Unit,
+    onSignOut: () -> Unit,
+    onLoadPortal: () -> Unit
+) {
+    Column(
+        modifier = Modifier
+            .fillMaxWidth()
+            .padding(8.dp),
+        horizontalAlignment = Alignment.CenterHorizontally,
```

Comment:
```suggestion
        horizontalAlignment = CenterHorizontally,
```

Reply:
I'll make this change, but Other samples are it this way too (Alignment.x), 
good to have an issue to update across all samples in future 
(similar to one below)